src/analysis/normed_space/star/spectrum.lean

```diff
@@ -0,0 +1,39 @@
+/-
+Copyright (c) 2022 Jireh Loreaux. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Jireh Loreaux
+-/
+import analysis.normed_space.star.basic
+import analysis.normed_space.spectrum
+
+/-! # Spectral properties in C⋆-algebras
+In this file, we establish various propreties related to the spectrum of elements in C⋆-algebras.
+-/
+
+open spectrum
+
+section unitary_spectrum
+
+variables
+{𝕜 : Type*} [normed_field 𝕜]
+{E : Type*} [normed_ring E] [star_ring E] [cstar_ring E]
+[normed_algebra 𝕜 E] [complete_space E] [nontrivial E]
+
+lemma unitary.spectrum_subset_circle (u : unitary E) :
+  spectrum 𝕜 (u : E) ⊆ { k : 𝕜 | ∥k∥ = 1 } :=
+begin
+  refine λ k hk, le_antisymm _ _,
+  { simpa only [cstar_ring.norm_coe_unitary u] using norm_le_norm_of_mem hk },
+  { have hcoe : (u : E) = (unitary.to_units u : E), from rfl,
```

Comment:
This `have` should be a lemma next to `to_units` - is it missing?

Reply:
[unitary.coe_to_units_apply](https://leanprover-community.github.io/mathlib_docs/algebra/star/unitary.html#unitary.coe_to_units_apply), it's there, I just wasn't thinking.